policy/modules/system/authlogin.if

```diff
@@ -859,9 +861,13 @@ interface(`auth_relabel_shadow',`
 interface(`auth_rw_shadow_lock',`
 	gen_require(`
 		type shadow_lock_t;
+		type etc_t;
 	')
 
-	rw_files_pattern($1, shadow_lock_t, shadow_lock_t)
+	files_etc_filetrans($1, shadow_lock_t, file, ".pwd.lock")
+	files_etc_filetrans($1, shadow_lock_t, file, "passwd.lock")
+	files_etc_filetrans($1, shadow_lock_t, file, "group.lock")
```

Comment:
Maybe these uses of 'files_etc_filetrans' belong in a new interface instead (something line auth_filetrans_shadow_lock).  These give permission to transition on file creation, but not to actually create the file.

Reply:
I agree.

Reply:
Done, and I did something similar for the other file transitions.  Let me know if this is what you were thinking of/if it's what you want.